<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalcSalario</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/iconoWin</normaloff>:/iconoWin</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblNombre">
            <property name="text">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="linEdNombre">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nombre&lt;/span&gt;: Ingresar el nombre del obrero&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="placeholderText">
             <string>Ingrese el nombre del obrero</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblHoras">
            <property name="text">
             <string>Horas</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxHoras">
            <property name="toolTip">
             <string>Ingrese el salario del obrero</string>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="toolTip">
          <string>Seleccione la jornada laboral</string>
         </property>
         <property name="title">
          <string>Jornada</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radButMatutina">
            <property name="text">
             <string>Matutina</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radButVespertina">
            <property name="text">
             <string>Vespertina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radButNocturna">
            <property name="text">
             <string>Nocturna</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushBcalcular">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Calcular el salario laboral</string>
            </property>
            <property name="text">
             <string>Calcular</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
          </font>
         </property>
         <property name="undoRedoEnabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="mnuAbrir"/>
    <addaction name="mnuNuevo"/>
    <addaction name="mnuGuardar"/>
    <addaction name="mnuGuardar_como"/>
    <addaction name="mnuCalcular"/>
    <addaction name="separator"/>
    <addaction name="mnuSalir"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mnuAbrir">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/abrir</normaloff>:/abrir</iconset>
   </property>
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="mnuGuardar_como">
   <property name="icon">
    <iconset theme="guardarComo" resource="Recursos.qrc">
     <normaloff>:/guardarComo</normaloff>:/guardarComo</iconset>
   </property>
   <property name="text">
    <string>Guardar &amp;como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="mnuCalcular">
   <property name="icon">
    <iconset theme="calcular" resource="Recursos.qrc">
     <normaloff>:/calcular</normaloff>:/calcular</iconset>
   </property>
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="mnuSalir">
   <property name="icon">
    <iconset theme="salir" resource="Recursos.qrc">
     <normaloff>:/salir</normaloff>:/salir</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="mnuNuevo">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/nuevo</normaloff>:/nuevo</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="mnuGuardar">
   <property name="icon">
    <iconset resource="Recursos.qrc">
     <normaloff>:/guardar</normaloff>:/guardar</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
